app/src/org/commcare/utils/Permissions.java

```diff
@@ -12,6 +12,12 @@
 import org.commcare.views.dialogs.DialogCreationHelpers;
 import org.javarosa.core.services.locale.Localization;
 
+import java.util.ArrayList;
```

Comment:
Thanks for making the code cleaner here, though just wanna confirm that no change in this class is relevant for Android 9 compatibility ?

Reply:
Correct, just a code cleanup